src/Backend/Core/Engine/TwigTemplate.php

```diff
@@ -30,14 +30,11 @@ class TwigTemplate extends BaseTwigTemplate
 {
     /**
      * The constructor will store the instance in the reference, preset some settings and map the custom modifiers.
-     *
-     * @param bool $addToReference Should the instance be added into the reference.
      */
-    public function __construct($addToReference = true)
+    public function __construct()
     {
-        if ($addToReference) {
```

Comment:
Hmmm, I should investigate further then, because removing this, I fixed everything.